Perfect number.py

```diff
@@ -0,0 +1,51 @@
+import math
+
+
+def integer_number(msg):
+    while True:
+        try:
+            v = int(input(msg))
+        except ValueError:
+            print("must be integer value")
+            continue
+        else:
+            return v
+            break
+
+
+print("-------------------------------------------------\n           ُPerfect Numbers")
+print("-------------------------------------------------")
+
+va = integer_number("Enter your value : ")
+list_of_perfect_numbers = [1]
+
+perfect = (va/2)+1
+
```

Comment:
why you are dividing `va/2 `

Reply:
maximum dividable of the n value is n/2
mean if n= 6 then n/2 = 3
so 1 + 2 + 3 = 6